c/xcsp/xcsp3_cpp_parser.license

```diff
@@ -0,0 +1,9 @@
+This file is part of the implementation translating XCSP3 benchmarks to C code:
+https://github.com/vsahil/XCSP3_to_C
+
+Copyright (c) 2008 Olivier ROUSSEL (olivier.roussel <at> cril.univ-artois.fr)
```

Comment:
The two `Copyright` lines can be removed, their content is also present in the `SPDX-FileCopyrightText` lines

Reply:
Yupp, thanks!